packages/git/src/browser/git-view-contribution.ts

```diff
@@ -104,6 +104,30 @@ export namespace GIT_COMMANDS {
     export const DISCARD_ALL = {
         id: 'git.discard.all'
     };
+    export const STASH = {
+        id: 'git.stash',
+        label: 'Stash'
```

Comment:
I am aware that VS Code does not have the ellipses after `Stash` either but we open the _Command Pallette_, I propose changing `Stash` to `Stash...`